src/main/java/org/jitsi/jigasi/transcription/LocalJsonTranscriptHandler.java

```diff
@@ -201,32 +201,20 @@ public void publish(ChatRoom room, TranscriptionResult result)
     {
         JSONObject eventObject = createJSONObject(result);
 
-        JSONObject encapsulatingObject = new JSONObject();
-        createEncapsulatingObject(encapsulatingObject, eventObject);
-
-        super.sendMessage(room, encapsulatingObject);
+        super.sendJsonMessage(room, eventObject);
     }
 
-    /**
-     * Create the JSON object will be send to the {@link ChatRoom}
-     *
-     * @param encapsulatingObject the json object which will be send
-     * @param transcriptResultObject the json object which will be added as
-     *                               payload
-     */
-    @SuppressWarnings("unchecked")
-    protected void createEncapsulatingObject(JSONObject encapsulatingObject,
-                                           JSONObject transcriptResultObject)
-    {
-        encapsulatingObject.put(JSON_KEY_TOPIC, JSON_VALUE_TOPIC);
-        encapsulatingObject.put(JSON_KEY_PAYLOAD, transcriptResultObject);
+    @Override
+    public void publish(ChatRoom chatRoom, TranslationResult result) {
+        //TODO: Send a JSON of different type to ChatRoom.
```

Comment:
I think it makes more sense to add it to this PR so that the enabling this does not ruin the "experience" of the transcription (e.g subtitles would break)